.github/workflows/tests.yml

```diff
@@ -21,7 +21,14 @@ jobs:
         uses: ./
         with:
             directory: gh-testing-research
-
+      - name: Assert log-stats file created
+        run: |
+            if test -f gh-testing-research/metadata/log_stats.json; then
+                echo "log-stats.json file exists"
```

Comment:
Nit: isn't it `log_stats.json`?

Reply:
It's not either, now that the command and default file are renamed (but yes, it should have been)